src/subdomains/supporting/payment/services/fee.service.ts

```diff
@@ -213,22 +213,27 @@ export class FeeService {
     const defaultFees = await this.getValidFees({ ...request, accountType });
 
     try {
-      return await this.calculateFee(defaultFees, request.from, request.to, request.allowBlockchainFeeFallback);
+      return await this.calculateFee(defaultFees, request.from, request.to, request.allowCachedBlockchainFee);
     } catch (e) {
       this.logger.error(`Fee exception, request: ${JSON.stringify(request)}`);
       throw e;
     }
   }
 
-  async getBlockchainFee(active: Active, allowFallback: boolean): Promise<number> {
+  async getBlockchainFee(active: Active, allowCached: boolean): Promise<number> {
     if (isAsset(active)) {
-      const fee = await this.blockchainFeeRepo.findOneCachedBy(`${active.id}`, {
+      const where = {
         asset: { id: active.id },
         updated: MoreThan(Util.minutesBefore(FeeValidityMinutes)),
-      });
-      if (!fee && !allowFallback) throw new Error(`No blockchain fee found for asset ${active.id}`);
+      };
+
+      const feeAmount = allowCached
+        ? await this.blockchainFeeRepo.findOneCachedBy(`${active.id}`, where).then((fee) => fee?.amount)
+        : await this.calculateBlockchainFee(active, false);
+      if (!feeAmount && !allowCached)
+        throw new Error(`No blockchain fee found for asset ${active.id} and allowCached: ${allowCached}`);
```

Comment:
It's not really useful to add `allowCached` to the message, it's always false in this case 😉